File: magic_8_ball/magic_8_ball.js
exports.handler = (event, context, callback) => {
    var question = event.Q;
    var answer = "";
    var randNum = Math.floor(Math.random() * 8 + 1);

    switch (randNum)
      {
        case 1:
            answer = "It is certain"
            break;
        case 2:
            answer = "Outlook good"
            break;
        case 3:
            answer = "You may rely on it"
            break;
        case 4:
            answer = "Ask again later"
            break;
        case 5:
            answer = "Concentrate and ask again"
            break;
        case 6:
            answer = "Reply hazy, try again"
            break;
        case 7:
            answer = "My reply is no"
            break;
        case 8:
            answer = "My sources say no"
            break;
        default:
            answer = "Something went wrong..."
            break;
    }

    callback(null, {
        "Answer": answer
    });
};

File: magic_8_ball/magic_8_ball.py
from random import seed, randint
import calendar
import time


seconds_since_epoch = calendar.timegm(time.gmtime())
def handler_name(event, context):
    seed(seconds_since_epoch)

    switch = {
        1: "Ask again later",
        2: "Concentrate and ask again",
        3: "It is certain",
        4: "My reply is no",
        5: "My sources say no",
        6: "Outlook good",
        7: "Reply hazy, try again",
        8: "Yes. No. Maybe? I dunno...",
        9: "You may rely on, it"
    }

    return switch[randint(1, len(switch))]
